documentation/overview.rst

```diff
@@ -1,5 +1,5 @@
 ***********************************
-Component System Overview
+Python Component System Intro
```

Comment:
Maybe we should change this title to something like "Python Component System (PCS) Intro"?  Otherwise we don't explicitly define the PCS acronym on this page before using it.

Reply:
Done.